### PR TITLE
Uyghur language translation updated, sura name added.

### PR DESCRIPTION
**Pull request by Sahran**
Uyghur translation file updated. booleans.xml and sura_names.xml added for Uyghur language.

### CODE REVIEW

**Comment by ahmedre** [comment]
jazakumAllah khairan! for the sura name updates, do you have a source for those just for future reference?

File: app/src/main/res/values-ug/sura_names.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <item>مائۇن</item>
+    <item>كەۋسەر</item>
+    <item>كافىرون</item>
+    <item>نەسر</item>
+    <item>مەسەد</item>
+    <item>ئىخلاس</item>
+    <item>فەلەق</item>
+    <item>ناس</item>
+  </string-array>
+
```
Comment (ahmedre) [inline app/src/main/res/values-ug/sura_names.xml:119]:
```suggestion
</resources>
```

**Comment by Sahran** [comment]
About Uyghur Sura Name Reference:
The sura names are from the Uyghur translation of the Quran by Muhammad Salih, published by the King Fahd Quran Printing Factory. (See details on page 657.)

PDF Version URL:
https://elkitab.org/wp-content/uploads/2018/01/ug_kuran_kerimning_terjimisi.pdf

Please check Uyghur sura names starting from page 651 to page 654.

Thanks.

**Comment by ahmedre** [comment]
awesome, jazakumAllah khairan!

**Pull request merged**